src/codegen/compile.ml

```diff
@@ -3588,7 +3618,7 @@ module Object = struct
   (* Returns a pointer to the object field (without following the field indirection) *)
   let idx_hash_raw env low_bound =
     let name = Printf.sprintf "obj_idx<%d>" low_bound  in
-    Func.share_code2 env name (("x", I32Type), ("hash", I32Type)) [I32Type] (fun env get_x get_hash ->
+    Func.share_code2 Func.Always env name (("x", I32Type), ("hash", I32Type)) [I32Type] (fun env get_x get_hash ->
```

Comment:
Just a note, not for this PR: This is probably a very frequently called function. Even if we would replace `Return` by loop exit, it is maybe too heavy-weight for inlining. A potential optimization for the future could be to use a (perfect) hash table lookup...

Reply:
To avoid confusion, Func.Always means always share, never inline. So I think we are in agreement.